README.md

```diff
@@ -4,10 +4,10 @@
 
 Setup | Status
 --- | ---
-Minimal | [![Build Status](https://jenkins1.klepsydra.com/buildStatus/icon?job=kpsr-core%2Fmaster)](https://jenkins1.klepsydra.com/view/Core/job/kpsr-core/job/master/)
-ZMQ | [![Build Status](https://jenkins1.klepsydra.com/buildStatus/icon?job=kpsr-zmq-core%2Fmaster)](https://jenkins1.klepsydra.com/view/Core/job/kpsr-zmq-core/job/master/)
-ROS 16.04 | [![Build Status](https://jenkins1.klepsydra.com/buildStatus/icon?job=kpsr-ros-core-16.04%2Fmaster)](https://jenkins1.klepsydra.com/view/Core/job/kpsr-ros-core-16.04/job/master/)
-ROS 18.04 | [![Build Status](https://jenkins1.klepsydra.com/buildStatus/icon?job=kpsr-ros-core-18.04%2Fmaster)](https://jenkins1.klepsydra.com/view/Core/job/kpsr-ros-core-18.04/job/master/)
+Minimal | [![Build Status](https://jenkins1.klepsydra.com/buildStatus/icon?job=kpsr-core%2Ffeature%252Flicense-header)](https://jenkins1.klepsydra.com/view/Core/job/kpsr-core/job/feature%2Flicense-header/)
+ZMQ | [![Build Status](https://jenkins1.klepsydra.com/buildStatus/icon?job=kpsr-zmq-core%2Ffeature%252Flicense-header)](https://jenkins1.klepsydra.com/view/Core/job/kpsr-zmq-core/job/feature%2Flicense-header/)
+ROS 16.04 | [![Build Status](https://jenkins1.klepsydra.com/buildStatus/icon?job=kpsr-ros-core-16.04%2Ffeature%252Flicense-header)](https://jenkins1.klepsydra.com/view/Core/job/kpsr-ros-core-16.04/job/feature%2Flicense-header/)
+ROS 18.04 | [![Build Status](https://jenkins1.klepsydra.com/buildStatus/icon?job=kpsr-ros-core-18.04%2Ffeature%252Flicense-header)](https://jenkins1.klepsydra.com/view/Core/job/kpsr-ros-core-18.04/job/feature%2Flicense-header/)
```

Comment:
This should probably not be merged into the master branch because it might cause the master branch to show the build status of this branch.

Reply:
This is post commit hook, but you are right....